controllers/update_status.go

```diff
@@ -624,9 +640,77 @@ func validateProcessGroup(ctx context.Context, r *FoundationDBClusterReconciler,
 	processGroupStatus.UpdateCondition(fdbv1beta2.PodFailing, failing, cluster.Status.ProcessGroups, processGroupStatus.ProcessGroupID)
 	processGroupStatus.UpdateCondition(fdbv1beta2.PodPending, false, cluster.Status.ProcessGroups, processGroupStatus.ProcessGroupID)
 
+	disableTaintFeature := false
+	for _, taintConfiguredKey := range cluster.Spec.AutomationOptions.Replacements.TaintReplacementOptions {
+		if *taintConfiguredKey.Key == "*" && *taintConfiguredKey.DurationInSeconds < 0 {
+			// Disable detecting or replacing taint key
+			disableTaintFeature = true
```

Comment:
Why is the taint feature disabled if the wildcard is set to a value less than zero? What if we have a more specific taint option set? Based on the design: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/docs/design/node_taint_eviction.md#proposed-design we should implement a boolean to enable/disable the taint feature.

Reply:
SRE can optionally enable taint for some key and disable it for another key. 
Using negative duration is more versatile and avoid having a lot of verbose enable-disable flags .

Reply:
see: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/d0a7c692c598e9e87ba1fa2019bee87c1edbb0b0/controllers/update_status.go#L678-L682